src/ex4/server/DB/migrations/20220622154325-auto-inc-items-id2.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.removeColumn("Items", "id");
+
+    await queryInterface.addColumn("Items", "id", {
+      type: Sequelize.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    });
+  },
+
+  async down(queryInterface, Sequelize) {
```

Comment:
here you only removed the column but it's not the opposite of what you did. you forgot to add the "old" id that you deleted at first place

* this code is not needed once you use only the changeColumn